bundles/trash.rst

```diff
@@ -117,6 +117,13 @@ with the correct data.
         }
     }
 
+.. Tip::
+
+    If `TrashItemHandler` is created outside of a Sulu sulu/skeleton application, for example in a separate bundle, the automatic autowire of the Trash bundle does not work.
```

Comment:
Would here go with the same links and wording like in the [symfony documentation](https://symfony.com/doc/5.4/form/create_custom_field_type.html):

> If you're using the [default services.yaml configuration](https://symfony.com/doc/5.4/service_container.html#service-container-services-load-example), this example will already work! Otherwise, [create a service](https://symfony.com/doc/5.4/service_container.html#service-container-creating-service) for this class and [tag it](https://symfony.com/doc/5.4/service_container/tags.html) with ``sulu_trash.store_trash_item_handler``.